### DISCUSSION TITLE
How to customize the server name?

### DISCUSSION THREAD

**Discussion by ghost** [AdguardTeam/AdGuardHome#3456] [Q&A]
### Prerequisites
- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### Problem Description

How to customize the server name?
![image](https://user-images.githubusercontent.com/51149477/129505907-367ee118-28fc-43c7-bc1c-49a0ec27962f.png)

adguardhome release: AdGuardHome_linux_arm64.tar.gz

**Comment by ainar-g** [comment]
According to [this article], you need to add a PTR record for your server's IP.  For example, if your server's IP in your network is `192.168.1.2`, try adding this to your custom rules:

```none
||2.1.168.192.in-addr.arpa^$dnsrewrite=NOERROR;PTR;myadguardhome.local
```

Where `myadguardhome.local` is the domain name you want it to show.

[this article]: https://pelegit.co.il/how-to-fix-default-server-unknown-nslookup/

**Reply by ghost** [reply]
Where are the custom rules?

is filters - DNS rewrites?
![image](https://user-images.githubusercontent.com/51149477/129668866-df10a05a-8942-4074-9b87-9f64eef28173.png)

**Reply by ghost** [reply]
I want to change "unknown" to "adguardhome"

**Reply by ainar-g** [reply]
No, custom filtering rules are a separate menu item under filters.

**Reply by ghost** [reply]
ok, thank you very much.

**Reply by ghost** [reply]
> According to [this article](https://pelegit.co.il/how-to-fix-default-server-unknown-nslookup/), you need to add a PTR record for your server's IP. For example, if your server's IP in your network is `192.168.1.2`, try adding this to your custom rules:
> 
> ```
> ||2.1.168.192.in-addr.arpa^$dnsrewrite=NOERROR;PTR;myadguardhome.local
> ```
> 
> Where `myadguardhome.local` is the domain name you want it to show.

What a great article! Thanks a lot!
